demo/screens/ListLands.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import {
+  ListView,
+  Image,
+  StyleSheet,
+  View,
+  Text,
+  Button,
+  TouchableHighlight,
+} from "react-native";
+import Land from "../components/presentational/Land";
+
+const rows = [
```

Comment:
And in the future we'll pull this from some external data source. But I like hard-coding it here for now.

Reply:
Ok